packages/hawtio/src/plugins/shared/workspace.ts

```diff
@@ -26,9 +28,25 @@ class Workspace implements IWorkspace {
   private pluginUpdateCounter?: number
   private treeWatchRegisterHandle?: Promise<number>
   private treeWatcherCounter?: number
+  private _errors: Error[] = []
```

Comment:
Why does this need to be `Error` class?  Can't we just use `string[]` with only error messages here?

Reply:
The benefit was the preservation of the actual cause of the error caught by adding it to the `cause` property of an Error.